backend/services/event_service.py

```
@@ -0,0 +1,71 @@
+            updated = True
+    if updated:
+        db.commit()
+        db.refresh(model_instance)
+    return updated
+
+def update_post_and_event(
+    db: Session,
+    post,
+    event,
+    post_data: dict,
+    event_data: dict
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)